### PR TITLE
Mock a DB service for "Acción" in add linix use cases

### PR DESCRIPTION
**Pull request by luisalejandrosalazar**

### CODE REVIEW

File: src/mocks/utils/initializeDataDB.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ import { buttonOptionsMock } from "@mocks/privileges/button/buttonOptionsMock.mo
 import { webOptionsMock } from "@src/mocks/privileges/web/webOptionsMock.mock";
```
Comment (LinconOssa) [inline src/mocks/utils/initializeDataDB.ts:8]: here you could use the alias @mocks as in the others.

Comment (luisalejandrosalazar) [reply src/mocks/utils/initializeDataDB.ts:8]: I already added an alias on line 11, isn't that correct?

**Review by LinconOssa — APPROVED** [review]
Meets requirements

File: src/pages/privileges/outlets/roles/types.ts [hunk 2]
```diff
@@ -81,3 +81,9 @@ export interface IFormAddRole {
   crediboardTasks: IOptionInitialiceForm;
   useCases: IOptionInitialiceForm;
 }
+
+export interface IActions {
```
Comment (carin91) [inline src/pages/privileges/outlets/roles/types.ts:85]: The actions are related to "Caso de uso linix", therefore, the IActions are a "linix use case" type. Relocate the interface IActions definition to linixUseCase/types.ts

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/pages/privileges/outlets/roles/types.ts (hunk 2 above)
Comment (luisalejandrosalazar) [reply src/pages/privileges/outlets/roles/types.ts:85]: Relocated

**Review by carin91 — APPROVED** [review]

**Pull request merged**